spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,15 @@
+
+require 'beaker-rspec/spec_helper'
+require 'beaker-rspec/helpers/serverspec'
+require 'beaker/puppet_install_helper'
+require 'beaker/module_install_helper'
+
+run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
```

Comment:
We should probably add this to our modulesync_config rather sooner than later.

Reply:
For the other modules you mean? If this is going to be a PDK compat module, we can't use modulesync as well.